### DISCUSSION TITLE
Genealogical relatedness instead of genetic relatedness

### DISCUSSION THREAD

**Discussion by fouerghi** [tskit-dev/tskit#2822] [Q&A]
Hi everyone, I simulated some microsats with SLiM and I also enabled remembering each individual and produced a tree sequence. I would like to take this tree sequence and calculate the genealogical relatedness between individuals (not just the nodes). I see a feature in tskit that allows to calculate the genetic_relatedness(), but I am interested in the genealogical relatedness as a feature. Does anyone know how I can calculate this using the tree sequence I have?

**Comment by petrelharp** [comment]
I'm closing this in favor of https://github.com/tskit-dev/pyslim/discussions/325
